ibazel.rb

```diff
@@ -0,0 +1,37 @@
+# Copyright 2018 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+class Ibazel < Formula
+  desc 'IBazel is a tool for building Bazel targets when source files change.'
+  homepage 'https://github.com/bazelbuild/bazel-watcher'
+  version '0.5.0'
+  url "https://codeload.github.com/bazelbuild/bazel-watcher/tar.gz/v0.5.0"
+  sha256 '6b727c60a807b56ebcf7b76c1002657bd93ad9199a2caa1c7ccd1a7eff4a4659'
+
+  bottle :unneeded
+
+  depends_on "bazel" => :install
```

Comment:
I think this depends on the bazel in homebrew core no? If so that's not what we would want.

Reply:
How would I specify the tap based version?

Reply:
I have no idea about this, but maybe it is `bazelbuild/tap/bazel` as this is also the `brew install` command that has be used to install bazel from this tap.